db/migrate/20161008183631_add_tables_for_url_refered_by_event_name_user_agent_res_height_and_width_and_ip.rb

```diff
@@ -0,0 +1,35 @@
+class AddTablesForUrlReferedByEventNameUserAgentResHeightAndWidthAndIp < ActiveRecord::Migration
+  def change
+    drop_table :requests
+
+    create_table :referred_bys do |t|
+      t.string :referred_by
+    end
+
+    create_table :urls do |t|
+      t.string :url
+    end
+
+    create_table :event_names do |t|
+      t.string :event_name
```

Comment:
i'd choose better naming conventions here in your database. be sure that the attributes in your tables don't match the table name itself. this becomes complicated/confusing very quickly